org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/CompletionItemDefaultsUtils.java

```diff
@@ -62,7 +62,10 @@ public static void process(CompletionResponse completionResponse, SharedSettings
 	 */
 	private static void setToMostCommonEditRange(List<CompletionItem> completionList,
 			CompletionItemDefaults itemDefaults) {
-		Map<Range, List<CompletionItem>> itemsByRange = completionList.stream()
+		Map<Range, List<CompletionItem>> itemsByRange = completionList
+				.stream()
+				.filter(item -> item.getTextEdit() != null && item.getTextEdit().isLeft()
+						&& item.getTextEdit().getLeft() != null)
```

Comment:
I think getLeft() is always non null if isLeft() == true

Reply:
good catch, fixed.